File: MES_EQUIPMENT/Models/WireBondingDto.cs
public class WireBondingDto
{
    public int Id { get; set; }                  // PK
    public int ProcessLogId { get; set; }     // FK: ProcessLog 참조 (숫자만)
    public double BondingTemp { get; set; }      // 본딩 온도
    public double BondingForce { get; set; }     // 본딩 힘
    public double UltrasonicPower { get; set; }  // 초음파 출력
    public double BondingTime { get; set; }      // 본딩 시간
    public double LoopHeight { get; set; }       // 루프 높이
    public double BallDiameter { get; set; }     // 볼 직경

    // ✅ 역직렬화: byte[] → 공정 DTO
    public static WireBondingDto FromBytes(byte[] payload)
    {
        int offset = 0;
        WireBondingDto dto = new WireBondingDto();

        dto.Id = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.ProcessLogId = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.BondingTemp = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.BondingForce = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.UltrasonicPower = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.BondingTime = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.LoopHeight = BitConverter.ToDouble(payload, offset); offset += 8;
        dto.BallDiameter = BitConverter.ToDouble(payload, offset); offset += 8;

        return dto;
    }
}

File: MES_SPRING/src/main/java/com/hm/mes_final_260106/dto/WireBondingDto.java
package com.hm.mes_final_260106.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class WireBondingDto {

    private Long id;                // PK: 공정 ID
    private Long productionLogId;  // pk: ProductionLog 참조 (숫자만)

    private double bondingTemp;     // 본딩 온도 (℃)
    private double bondingForce;    // 본딩 압력 (gf)
    private double ultrasonicPower; // 초음파 출력 (mW)
    private double bondingTime;     // 본딩 시간 (ms)
    private double loopHeight;      // 루프 높이 (μm)
    private double ballDiameter;    // 볼 직경 (μm)
}
///1111111111